Design watersurfaces

ToonHub opened the pull request:
I slightly modified the structure of the table with assessments. I used `any_habitat` instead of `is_habitat`. See documentation in rendered bookdown below. 

[mhq_design_evaluation.zip](https://github.com/inbo/n2khab-mhq-design/files/7236422/mhq_design_evaluation.zip)

To do:

- [x] Add `MHQ2190_aq` to `scheme_types` table in `n2khab` package
- [x] Add final version of tables to repo's `n2khab-samplingframes` and `n2khab-sample-admin`

florisvdh commented:
I'll be able to review beginning of next week. Can you post the latest compiled version?

ToonHub commented:
Here the latest version:
[mhq_design_evaluation.zip](https://github.com/inbo/n2khab-mhq-design/files/7276294/mhq_design_evaluation.zip)

ToonHub commented on habitatwatersurfaces/src/20_admin_standingwater.Rmd:
```diff
@@ -0,0 +1,1043 @@
+
+When sampling units does not contain the target type new sampling units have to be selected based on the grts-ranking. 
+In some cases it was decided not to assess a new sampling unit that should have been selected according to the grts-ranking. 
+This because the new sampling unit was located in a region in which already sufficient sampling units were measured.
```
According to the field worker and indeed possibly biased. These sampling units are considered as not being assessed (we do not know if the habitat type is present) and can therefore be selected for the next monitoring cycle.

ToonHub commented on habitatwatersurfaces/src/20_admin_standingwater.Rmd:
```diff
@@ -0,0 +1,1043 @@
+
+For 2190 the watersurface code was not recorded during measurement.
+However the sampling units almost fully match with the watersurfaces.
+One polygon (WV_DHE_020) overlaps with 5 watersurface polygons: for now we assume that watersurface WVLBRD0089 is the correct watersurface: TO BE CHECKED!
```
Yes

ToonHub commented on habitatwatersurfaces/src/20_admin_standingwater.Rmd:
```diff
@@ -0,0 +1,1043 @@
+
++ The target type of the original sampling unit should be the same as the observed type in the newly selected sampling unit. This is not the case for one sampling unit.
++ The table also provides the distance between reference points. 
++ For one sampling unit it could not be identified which sampling unit it replaces (NOD_011: to be checked!)
```
yes

florisvdh commented on habitatwatersurfaces/src/index.Rmd:
```diff
@@ -0,0 +1,79 @@
+)
+
+# remotes::install_github("inbo/n2khab",
+#                         ref = "dev_nextrelease")
```
When using `renv`, note that you should add `renv::restore()` in the setup chunk, in order to enforce the `renv`-recorded package versions when running the R code.

ToonHub commented on habitatwatersurfaces/src/20_admin_standingwater.Rmd:
```diff
@@ -0,0 +1,1043 @@
+In this case the admin data contains two records for one watersurface, one for each of the original sampling units.
+However, both records refer to the same assessment and can be aggregated to one record.
+In this aggregation we set grts_ranking_draw to the minimum value of grts_ranking_draw for both original sampling units.
+After aggregation we recalculate the grts_ranking based on the watersurface polygon using st_point_on_surface. 
```
Good point. I modified this so that we keep the original reference point with the lowest grts_ranking.

ToonHub commented on habitatwatersurfaces/src/20_admin_standingwater.Rmd:
````diff
@@ -0,0 +1,1043 @@
+write_vc(watersurfaces_refpoints, "watersurfaces_refpoints", "../output", sorting = "polygon_id", strict = FALSE)
+```
+
++ watersurfaces_refpoints (git2rdata object): defines the reference point for the selected watersurfaces in the watersurfaces data source and grts-ranking for this refpoint
````
It is the original sampling frame. I changed this in the text.

ToonHub commented:
@florisvdh: thanks for the review! 

I added some commits to adress your comments. I also adressed your main comment and removed `scheme` from the table `watersurfaces_populationunits`.

You can find the latest rendered report below


[mhq_design_evaluation.zip](https://github.com/inbo/n2khab-mhq-design/files/7330375/mhq_design_evaluation.zip)
